schema.sql

```diff
@@ -33,4 +33,27 @@ ALTER TABLE animals
 ALTER TABLE animals 
     ADD foreign key (species_id) references species(id);
 ALTER TABLE animals 
-    ADD foreign key (owner_id) references owners(id);
\ No newline at end of file
+    ADD foreign key (owner_id) references owners(id);
+
+-- create vets table to the database
+CREATE TABLE vets (
+    id bigserial PRIMARY KEY,
```

Comment:
[ OPTIONAL] 
- Great job working on this milestone. You can consider using `int` datatype. `bigserial` will occupy more memory just for storing primary key id